repair/row_level.cc

```diff
@@ -3218,13 +3218,22 @@ future<> repair_service::start() {
 }
 
 future<> repair_service::stop() {
+  try {
+    rlogger.debug("Stopping repair task module");
     co_await _repair_module->stop();
+    rlogger.debug("Waiting on load_history_done");
     co_await std::move(_load_history_done);
+    rlogger.debug("Uninitializing messaging service handlers");
     co_await uninit_ms_handlers();
     if (this_shard_id() == 0) {
+        rlogger.debug("Unregistering gossiper helper");
```

Comment:
This amount of logging seems excessive to me. This is fine when you want to debug something, but in production code, this seems too much.

Reply:
See https://github.com/scylladb/seastar/blob/7cd0cabfcb6662dba0ad50aeb07a69b0ad20a786/tests/unit/coroutines_test.cc#L75-L78